engine/src/gui/text_area.cpp

```diff
@@ -208,85 +204,80 @@ void TextArea::RenderTextItem(TextAreaItem *current, int level) {
         if (count < top_item_number) {
             count++;
         } else {
-            new_y = ycoord[5] - (text_spacing * (count + 1 - top_item_number)) + horizontal_spacer;
-            new_x = xcoord[5] + (horizontal_per_level * (level - 1));
+            float new_x = 0;
```

Comment:
Why separate to two lines?

Reply:
Oh. Duh. I could declare `new_x` and `new_y` and initialize them to their final values on one line each, couldn't I? Good catch